blog/2022-06-16-inlong-sort-etl.md

```diff
@@ -19,9 +18,9 @@ First of all, based on Apache Flink SQL, there are mainly the following consider
 
 **Note**:  for all codes of this scheme, please refer to [Apache inlong sort]（ https://github.com/apache/incubator-inlong/tree/master/inlong-sort ）Module, which can be downloaded and used in the upcoming version 1.2.0.
```

Comment:
for all codes of this scheme -> For all codes of this scheme

Reply:
done.